build.rs

```diff
@@ -20,10 +21,20 @@ fn main() -> std::io::Result<()> {
     std::env::set_var("PROTOC", protoc_bin_vendored::protoc_bin_path().unwrap());
 
     if let Err(err) = get_and_build_protos(
-        &["https://raw.githubusercontent.com/cloudevents/spec/main/cloudevents/formats/cloudevents.proto", 
-        "https://raw.githubusercontent.com/googleapis/googleapis/master/google/rpc/status.proto"]
+        &[
+            // cloudevent proto definitions
+            "https://raw.githubusercontent.com/cloudevents/spec/main/cloudevents/formats/cloudevents.proto", 
+
+            // uProtocol-project proto definitions
+            "https://raw.githubusercontent.com/eclipse-uprotocol/uprotocol-core-api/uprotocol-core-api-1.5.3/src/main/proto/uuid.proto",
+            "https://raw.githubusercontent.com/eclipse-uprotocol/uprotocol-core-api/uprotocol-core-api-1.5.3/src/main/proto/uri.proto",
+            "https://raw.githubusercontent.com/eclipse-uprotocol/uprotocol-core-api/uprotocol-core-api-1.5.3/src/main/proto/uattributes.proto",
+            "https://raw.githubusercontent.com/eclipse-uprotocol/uprotocol-core-api/uprotocol-core-api-1.5.3/src/main/proto/upayload.proto",
+            "https://raw.githubusercontent.com/eclipse-uprotocol/uprotocol-core-api/uprotocol-core-api-1.5.3/src/main/proto/umessage.proto",
+            "https://raw.githubusercontent.com/eclipse-uprotocol/uprotocol-core-api/uprotocol-core-api-1.5.3/src/main/proto/ustatus.proto",
```

Comment:
in order to be able to consistently use the types generated from these protos it would be preferable to also include the other core API protos:
* file.proto
* uprotocol_options.proto
* udiscovery/v3/udiscovery.proto
* usubscription/v3/usubscription.proto
* utwin/v1/utwin.proto

Reply:
I'm pushing to conan now the build recipe for c++ of uprotocol-core-api, we should do the same to crates.io so you don't have to manually pull in these protos.

Reply:
done

Reply:
I guess that would make sense. However, currently it seems that the uprotocol-core-api repo only contains archives of the whole repository and IMHO it would be more appropriate to create *release* artifacts instead, i.e. archives which only contain the proto files ...

Reply:
> @AnotherDaniel , if we go with @sophokles73 proposal [eclipse-uprotocol/uprotocol-core-api#76](https://github.com/eclipse-uprotocol/uprotocol-core-api/issues/76) (uprotocol-core-api only contains the protos that each language has to pull and build and push to their respective central repositories) then wouldn't it be better to pull the tar..gz, unpack and build all of them?

a) which .tar.gz - am I missing something somewhere?
b) going via an archive would pull in an additional functional dependency, and additional code into the build file; can be done, but what would be the benefit?